sdk/identity/identity/samples-dev/clientSecretCredential.ts

```diff
@@ -1,16 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { ClientSecretCredential } from "@azure/identity";
+/**
+ * @summary Authenticates with a client and a client's secret.
+ */
+
+import { ClientSecretCredential } from "../src";
 import { KeyClient } from "@azure/keyvault-keys";
```

Comment:
Two comments here:

- We need to still be importing from `@azure/identity` here. Adding it to the `paths` entry in tsconfig should enable this, so you don't need to import from source for that to resolve (as you can think of `paths` as adding a manual module resolution entry to the TypeScript compiler).
- We will also need an entry for `@azure/keyvault-keys`. I never thought to use `dependencyOverrides` for this purpose, but it should totally work if we also add a `paths` entry for it! I think something like `"@azure/keyvault-keys": ["../../keyvault/keyvault-keys/src/index"]` should work nicely.

Reply:
So, I updated the references to `@azure/identity` on the samples, but `"@azure/keyvault-keys": ["../../keyvault/keyvault-keys/src/index"]` didn’t work. We talked over teams and we realized that adding keyvault on paths would alter significantly how the `dist` folder appears (it grabs a large part of the repository structure, to group these two projects under the same root).